RecoMET/METFilters/plugins/EcalDeadCellTriggerPrimitiveFilter.cc

```diff
@@ -158,6 +162,9 @@ class EcalDeadCellTriggerPrimitiveFilter : public edm::EDFilter {
 
   bool useTPmethod_, useHITmethod_;
 
+  bool exclusivelykTPSaturated_;
+  bool considerkTPSaturated_;
```

Comment:
I'm not sure why these two are added: there are not used in the class methods.
Please clarify.